app/config/config.yml

```diff
@@ -199,3 +199,42 @@ scheb_two_factor:
 kphoen_rulerz:
     executors:
         doctrine: true
+
+fos_elastica:
+    clients:
+        default: { host: localhost, port: 9200 }
+    serializer:
+        callback_class: FOS\ElasticaBundle\Serializer\Callback
+        serializer: serializer
+    indexes:
+        wallabag:
+            client: default
+            settings:
+                index:
+                    analysis:
+                        analyzer:
+                            custom_french_analyzer:
+                                type: custom
+                                tokenizer: letter
+                                filter: ["asciifolding", "lowercase", "french_stem", "stop_fr"]
```

Comment:
It can help yes, 
you could chose to support some strong elision on a list of language (the one already in ES core are a [good starting point](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-lang-analyzer.html)), and search on those on a content basis.

Have a quick look at this to learn more about the recommended config: https://www.elastic.co/guide/en/elasticsearch/guide/current/mixed-lang-fields.html#_use_n_grams